.travis.yml

```diff
@@ -30,3 +30,8 @@ deploy:
   acl: public_read
   on:
     branch: master
+  # deploy services
+  provider: script
+  script: bash services/deploy.sh prod
```

Comment:
Just take a look at this not sure if it helps.
https://docs.travis-ci.com/user/build-stages/